snippets/deepClone.md

```diff
@@ -8,15 +8,25 @@ Use `Object.keys()` and `Array.forEach()` to determine which key-value pairs nee
 
 ```js
 const deepClone = obj => {
-  let clone = Object.assign({}, obj);
-  Object.keys(clone).forEach(
-    key => (clone[key] = typeof obj[key] === 'object' ? deepClone(obj[key]) : obj[key])
-  );
-  return clone;
```

Comment:
Should have been the only changed line.

`return Array.isArray(obj) ? (clone.length = obj.length) && Array.from(clone) : obj;`

Reply:
From the contribution guidelines:
>Always use soft tabs (2 spaces), never hard tabs.